app/main.py

```diff
@@ -1,2 +1,75 @@
+from typing import Any, Hashable
+from copy import deepcopy
+
+
 class Dictionary:
-    pass
+    class Node:
+        def __init__(self, key: Hashable, value: Any, my_hash: int) -> None:
+            self.key = key
+            self.value = value
+            self.my_hash = my_hash
+
+        def __repr__(self) -> str:
+            return f"Node(key={self.key}, value={self.value})"
+
+    def __init__(self) -> None:
+        self.__capacity = 8
+        self.__load_factor = 2 / 3
+        self.__size = 0
+        self.__arr = [0 for _ in range(self.__capacity)]
```

Comment:
why not call this structure a hash_table? also it is better to fill it up with None values. Also it will be beneficial to annotate that hash table is a list of Nodes or None values